src/main/java/messenger/ImageMessengerNotification.java

```diff
@@ -0,0 +1,63 @@
+package messenger;
+
+import notification.Notification;
+
+import java.time.LocalDateTime;
+
+/**
+ * Created by annterina on 11.11.17.
+ */
+public class ImageMessengerNotification implements Notification {
+
+    private String senderId;
+    private String receiverId;
+    private String senderName;
+    private String receiverName;
+    private LocalDateTime sendDate;
+    private String imageName;
```

Comment:
If you are storing three fields containing information from `MultipartFile` why not just store single field e.g. `private MultipartFile image`. If you want to provide getters for image name or actual content you can just use `MultipartFile` methods in them to get that information. This also reduces the number of fields in the whole class.

Reply:
Ok!